homeassistant/components/zwave_js/api.py

```diff
@@ -369,12 +372,7 @@ async def websocket_refresh_node_info(
 ) -> None:
     """Re-interview a node."""
     node_id = msg[NODE_ID]
-    controller = client.driver.controller
-    node = controller.nodes.get(node_id)
-
-    if node is None:
-        connection.send_error(msg[ID], ERR_NOT_FOUND, f"Node {node_id} not found")
-        return
+    node = client.driver.controller.nodes[node_id]
```

Comment:
Is the plan to blow up with KeyError if the node is missing? Ie this is just a developer mistake?

Reply:
not sure what I was trying to do here but I added node validation back in